app/src/main/java/org/mozilla/fenix/settings/quicksettings/QuickSettingsController.kt

```diff
@@ -199,21 +198,21 @@ class DefaultQuickSettingsController(
     }
 
     override fun handleBlockedItemsClicked() {
-        dismiss.invoke()
+        navController().popBackStack()
 
         val state = quickSettingsStore.state.trackingProtectionState
-        val directions = QuickSettingsSheetDialogFragmentDirections
+        val directions = NavGraphDirections
```

Comment:
Did this need to be changed?

Reply:
This one it's an improvement, as this action doesn't belong to `QuickSettingsSheetDialogFragment` it's a global action `actionGlobalTrackingProtectionPanelDialogFragment`, using `NavGraphDirections` make it straightforward.